terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/el/ElTest.java

```diff
@@ -403,6 +403,272 @@ public void test06_Query_Display() {
     }
 
     @Test
+    public void test06_09_NestedJavaBean() {
+        driver.get(applicationContextUrl);
+        driver.findElement(By.id("EL")).click();

```

Comment:
Probably, two step of above are unnecessary. it perform by `@Before` method of parent class.